FLEDGE.md

```diff
@@ -195,6 +195,8 @@ The `ads` list contains the various ads that the interest group might show.  Eac
 
 The `adComponents` field contains the various ad components (or "products") that can be used to construct ["Ads Composed of Multiple Pieces"](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#34-ads-composed-of-multiple-pieces)). Similar to the `ads` field, each entry is an object that includes a `renderURL` and optional `adRenderId`, and `metadata` fields. Thanks to `ads` and `adComponents` being separate fields, the buyer is able to update the `ads` field via the `updateURL` without losing `adComponents` stored in the interest group.
 
+The `auctionServerRequestFlags` field is only used for auctions run on an auction server. This field contains a list of enumerated values that change what data is sent in the auction blob (). The `omit-ads` enumeration causes the request to omit the `ads` and `adComponents` fields for this interest group from the auction blob. The `include-full-ads` field causes the request to include the full ad object in place of anywhere in the request where a plain `adRenderId` would have been sent (such as in the `ads` and `adComponents` fields as well as `prevWins`. Note that `include-full-ads` is not compatible with the auction server, so this mode is only for debugging.
```

Comment:
Is this PR ready for review?  If so, the "()" looks like we meant to put something in there.  Are the "omit-ads" and "include-full-ads" the possible enumeration values?  if so we might want to list them out or change the wording to refer to them as possible enumerated values, rather than referring to "omit-ads" as an enumeration and "include-full-ads" as a field perhaps?  Sorry if I'm misinterpreting or reviewing too early.

Reply:
It was meant to be ready for review.
Thanks for your comments. I have removed the extraneous parentheses and clarified the wording.